example/flutter/android/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,44 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Can you leave comments on what lines of the file were added/modified from the generated file? That way it will be easier to deal with conflicts etc. if the Flutter project needs to be updated in the future.